modules/ROOT/pages/design-add-api-dependency.adoc

```diff
@@ -3,13 +3,14 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In projects for API specifications or RAML API fragments, you can add dependencies of either API fragments (RAML) or
-rulesets. Because the dependencies are referenced in Anypoint Exchange, you cannot directly edit them in your projects.
+In projects for API specifications or API fragments, you can add dependencies of either RAML fragments, OAS components, JSON schemas, or rulesets. Because the dependencies are referenced in Anypoint Exchange, you cannot directly edit them in your projects.
 
 If you export your project or someone downloads your project as RAML from Exchange, the resulting ZIP file contains the referenced dependencies.
 
 CAUTION: You must not edit the referenced dependencies in an editor other than the API Designer editor. If you export your project to work on it in another editor, edit the referenced dependencies, 
-and then try to import the project back into the text editor, the import fails.
+and then try to import the project back into the text editor, the import fails. +
+OAS 3.0 fragments are not supported in Studio.
```

Comment:
Studio does not support OAS 3.0 fragments.

For passive voice, but not a huge deal.